gems/smithy-client/lib/smithy-client/base.rb

```diff
@@ -9,8 +9,8 @@ class Base
       # @api private
       @plugins = PluginList.new(
         [
-          Plugins::Endpoint
-          # Plugins::NetHttp,
+          Plugins::Endpoint,
+          Plugins::NetHTTP
```

Comment:
Nit: this would also need to be determined by the api.

Also - how are we planning on mixing http1 and http2.  These plugins are at the client level, not at the operation level?

Reply:
Ah yeah. I think for now we can keep it default for the sake of making it work and remove that bit later.  We need to think about how clients with both http and http 2 operations will work. I think we can register both plugins but the handlers will only register for given operations.